crates/loro-internal/src/diff_calc/tree.rs

```diff
@@ -506,6 +506,11 @@ impl TreeCacheForDiff {
         effected
     }
 
+    pub(crate) fn apply_gc(&mut self, node: MoveLamportAndID) {
```

Comment:
Should pick a better name here, like `init_tree_with_trimmed_version`

Maybe it's better to receive all the tree nodes at once from the gc_state and initialize the whole structure here.
So, we can assert that it was empty.